Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -35,6 +35,16 @@ internal static class ProfileManager
 
         private static string _processName = System.Environment.ExpandEnvironmentVariables("%SystemDrive%\\msvsmon\\profiler\\VSStandardCollector.Dev14.exe");
 
+        static ProfileManager()
+        {
```

Comment:
@puneet-gupta plz remove the assignment here: private static readonly TimeSpan _profilingIisTimeout = TimeSpan.FromMinutes(3);
and add that logic to the static constructor.